app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_post_comment_counter
+    post.comments_counter += 1
+    post.save
+  end
+end
```

Comment:
- [ ] As mentioned above please use `after_save` for comment as well and also make it private.
`after_save : update_post_comment_counter`